snap/hooks/install

```diff
@@ -36,6 +36,12 @@ for service in security-file-token-provider security-proxy-setup security-secret
     fi
 done
 
+# install the template config yaml file for securing Kong's admin APIs in security-secretstore-setup service
+if [ ! -f "$SNAP_DATA/config/security-secretstore-setup/res/kong-admin-config.template.yml" ]; then
+    cp "$SNAP/config/security-secretstore-setup/res/kong-admin-config.template.yml" \
+        "$SNAP_DATA/config/security-secretstore-setup/res/kong-admin-config.template.yml"
+fi
+
```

Comment:
How to update this file if it is being conditionally replaced?

Reply:
not sure what is conditionally replaced? I don't think we have that capability yet.

Reply:
please describe and elaborate the use case you are thinking for "conditionally replaced"

Reply:
The `cp` command is skipped if `$SNAP_DATA/config/security-secretstore-setup/res/kong-admin-config.template.yml` already exists.  If it is a template, why not overwrite it every time?

Reply:
oh, that, i just refer to the same logic above when dealing withe the toml configuration file, too. see line 14 above. just to keep it consistent with that.